### ISSUE TITLE
[Feature Request]: Async loading of `editor.js`

### ISSUE THREAD

**Issue by Ddystopia** [siku2/rust-monaco#53]
Hello,

currently no rust code is executing before `editor.js` loaded, which means blank page. Is there a possibility to load it only when `yew::CodeEditor` is mounted, while not blocking other rust code?

**Comment by ctron** [comment]
So what would happen if the Rust code would try to call JS functions before the JS file is loaded?

**Comment by Ddystopia** [comment]
> So what would happen if the Rust code would try to call JS functions before the JS file is loaded?

I guess it could block until js file is loaded

**Comment by ctron** [comment]
And through which mechanism would that work?

**Comment by Ddystopia** [comment]
I don't know how should it work, but it would be very useful.

**Comment by ctron** [comment]
Yes, however I believe it's technically not possible. Unless you had an idea on how to achieve this.
